controllers/lmes/lmevaljob_controller.go

```diff
@@ -1361,6 +1365,11 @@ func generateArgs(svcOpts *serviceOptions, job *lmesv1alpha1.LMEvalJob, log logr
 		}
 	}
 
+	// --confirm_run_unsafe_code
+	if job.Spec.AllowCodeExecution != nil && *job.Spec.AllowCodeExecution {
+		cmds = append(cmds, "--confirm_run_unsafe_code")
```

Comment:
Should we add explicit logging here? We can add to other flags too, in retrospect, but in another PR.

```go
log.Info("Enabling unsafe code execution for LMEval job", "jobName", job.Name,  "namespace", job.Namespace)
```